.github/ISSUE_TEMPLATE/design-decision.md

```diff
@@ -10,6 +10,8 @@ assignees: ''
 ### Context and Problem Statement
 
 
+### Use-cases
```

Comment:
I would prefer "Use Cases" which is the more common form.

Reply:
Fine with me, changed.